tardis/plasma/tests/test_complete_plasmas.py

```diff
@@ -1 +1,139 @@
+import os
 
+import pandas as pd
+import pytest
+from pandas.util import testing as pdt
+from numpy.testing import assert_almost_equal
+from tardis.io.config_reader import Configuration
+from tardis.simulation import Simulation
+
+
+setupI = {
```

Comment:
@yeganer Did you mean , that like for example , `ionization` : `nlte` , that the values will be same, irrespective of other plasma config options?